app/components/DisplayCarFacts/DisplayCarFacts.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const DisplayCarFacts = ({carData}) =>{
+  return (
+    <div>
+      <h2>{carData.make} {carData.model}</h2>
+      <p>Years Manufactured: {carData.years.map((year, i) =>{
+        if(i === carData.years.length - 1){
+          return(
+            <span>{year} </span>
+          )
+        } else {
+          return(
+            <span>{year}, </span>
+          )
+        }
```

Comment:
If you'd like to shorten this up a bit, I can show you a regex that can replace the 'if' block. Otherwise, looking good my dude.